src/orchard/clojuredocs.clj

```diff
@@ -34,6 +40,18 @@
        (reset! cache))
   true)
 
+(defn test-to-access-url [^String url]
```

Comment:
I guess this can be `test-remote-url`. Btw, are we certain this is not going to blow up if someone passes a local file URL instead?

Reply:
You're right. I fixed to validate the format of `url`.